sklearn/utils/sparsefuncs_fast.pyx

```diff
@@ -24,30 +24,30 @@ ctypedef cnp.float64_t DOUBLE
 
 
 def csr_row_norms(X):
-    """L2 norm of each row in CSR matrix X."""
+    """Squared L2 norm of each row in CSR matrix X."""
     if X.dtype not in [np.float32, np.float64]:
         X = X.astype(np.float64)
-    return _csr_row_norms(X.data, X.shape, X.indices, X.indptr)
+    return _csr_row_norms(X.data, X.indices, X.indptr).base
```

Comment:
As discussed IRL with @jjerphan and @ArturoAmorQ, it might be more appropriate to return a numpy array in `_csr_row_norms` directly rather than here. WDYT?